src/scripts/main.js

```diff
@@ -21,7 +21,34 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key1 in food) {
+    const ol = document.createElement('ol');
+
+    ol.textContent = key1;
+    document.body.appendChild(ol);
+
+    if (typeof food[key1] === 'object') {
```

Comment:
The condition `typeof food[key1] === 'object'` should be `typeof data[key1] === 'object'` to correctly reference the `data` parameter.